src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -5623,6 +5623,13 @@ private static string DescribeLoop(RegexNode node, RegexMethod rm)
             return style + bounds;
         }
 
+        private static string ToHexStringNoDashes(byte[] bytes) =>
+#if NETCOREAPP
+            Convert.ToHexString(bytes);
```

Comment:
This is ok, but it's effectively dead code as the only asset we use is the netstandard2.0 one. We could just suppress the warning.